Chapter1/BMI_Calculator.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter1
+{
+
+    public static class BMICalculator
```

Comment:
Shouldn't have been included in hw

Reply:
We do not need the class or the static part ?

Reply:
I think this is a V1 Chap1 homework not V2?

Reply:
nope. this is v2 homework. See https://github.com/csinn/CSharp-From-Zero-To-Hero-v2/wiki/C%23-Keywords-and-User-Input

Reply:
the array_functions is not the right one. that is challenge 3